app/views/unattended/provisioning_templates/snippet/kickstart_kernel_options.erb

```diff
@@ -29,7 +29,11 @@ snippet: true
 
   # tell Anaconda what to pass off to kickstart server
   # both current and legacy syntax provided
-  options.push("kssendmac", "ks.sendmac", "inst.ks.sendmac")
+  if (is_fedora && os_major >= 33) || (rhel_compatible && os_major >= 9)
+    options.push("inst.ks=#{foreman_url('provision')}", "inst.ks.sendmac")
+  else
+    options.push("ks=#{foreman_url('provision')}", "kssendmac", "ks.sendmac")  
```

Comment:
I'm not sure if I would remove `inst.ks.sendmac` here, given that it was present previously. Keeping it in feels safer, but no objection if you think it's safe.

Reply:
The correct (and required) argument is `inst.ks.sendmac` both of the other forms were deprecated. Thanks.